Source/ChocolateyGui.Common.Windows/ViewModels/Items/PackageViewModel.cs

```diff
@@ -416,6 +420,12 @@ public bool IsPackageSizeAvailable
             get { return PackageSize != -1; }
         }
 
+        public void ShowArguments()
+        {
+            var decryptedArguments = _packageArgumentsService.Decrypt(Id, Version.ToString());
+            MessageBox.Show(decryptedArguments);

```

Comment:
@punker76 did you have any suggestions on the best way to display the returned data here?